packages/fasm/KagamiBuild

```diff
@@ -0,0 +1,15 @@
+# Description: An assembler for x86 processors.
+# URL:         https://flatassembler.net
+# Maintainer:  ixavoz, ixavoz at disroot dot com
+# Section:     devel
+name=fasm
+version=1.73.22
+release=1
+source=("https://flatassembler.net/fasm-$version.tgz")
+build()
+{
+    # I don't know how to write shell scripts.
+    "$SRC"/fasm/fasm.x64 "$SRC"/fasm/source/Linux/fasm.asm
```

Comment:
replace spaces with tabs

Reply:
I prefer spaces to tabs. Fix for consistency to the rest of KagamiBuild files anyway.